openapi-specs/schema.json

```diff
@@ -849,6 +849,31 @@
         }
       }
     },
+    "ObjectTtlConfig":{
+      "description": "Configuration of objects' time-to-live",
+      "properties": {
+         "enabled": {
+          "description": "Whether or not object ttl is enabled for this collection (default: `false`).",
+          "type": "boolean",
+          "x-omitempty": false
+        },
+        "defaultTtl": {
```

Comment:
Can we omit this for custom properties? This makes parsing in the clients a lot easier